lib/src/model/analysis/retro_controller.dart

```diff
@@ -163,14 +152,13 @@ class RetroController extends _$RetroController with EngineEvaluationMixin {
         feedback: RetroFeedback.findMove,
         mainlinePath: _root.mainlinePath,
         pov: options.initialSide,
-        currentNode: RetroCurrentNode.fromNode(_root.view),
```

Comment:
Avoid using root.view when possible.

Reply:
Makes sense, I think the missing conversion from view back to mutable node was the reason why I used views here